### DISCUSSION TITLE
grblcontrol issues

### DISCUSSION THREAD

**Discussion by submicron13** [meerk40t/meerk40t#2484] [Q&A] [answered]
I can't seem to get Lightburn to recognize my K40. When I do the detect device it shows up as: LihuiyuDevice not supported lightburn.

I tried using grbl-lpc and it doesn't seem to do anything.

What am I doing wrong? 0.9.4 Meerk40t.

**Comment by submicron13** [comment]
I'm able to connect, and issue commands via telnet now. I can't get movement commands to work though. Commands entered into telnet are mirrored in lightburn, but nothing happens.

**Comment by jpirnay** [comment, marked as answer]
Fixed in 0.9.5b2
